GalaxyZooTouchTable/GalaxyZooTouchTable/Converters/ObjectToTypeStringConverter.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Globalization;
+using System.Windows.Data;
+
+namespace GalaxyZooTouchTable.Converters
+{
+    public class ObjectToTypeStringConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (value != null)
+            {
+                return value.GetType().Name;
+            }
+            return null;
+        }
+
+        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            throw new NotImplementedException();
```

Comment:
Should this be implemented?

Reply:
It's unnecessary to have a `ConvertBack` method for this converter, but the interface requires a method to be defined. This is often the practice to have a `NotImplementedException` as a placeholder.

Reply:
I figured as much, just wanted to make sure you weren't leaving something out by accident.